### DISCUSSION TITLE
Dual licensing 在开源盈利中的作用

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#87] [日常讨论]
Sachin Neravath 分享的关于他利用开源进行获利的经验：

https://twitter.com/SachinNeravath/status/1524736567377424389

**Comment by johnnychen94** [comment]
有个困惑，关于双 License，我理解的一直都是企业商用协议+普通用户开源协议的混合版本。但是最近看到一些 Apache/MIT, GPL/MIT 的混合 license，这里面是有什么讲究么？

比如说 https://github.com/NVIDIA/open-gpu-kernel-modules 采用的好像是 GPL/MIT

**Comment by wangleo61** [comment]
我的理解是这种双license的情况有两种：


1．  本身的开源项目在不同的场景下适配不同的许可，比如企业客户适配GPL, 学生学习目的的适配MIT

2． 另外一种情况是开源的项目的不同的模块的许可证可以是不同的，以G LibC举例，贡献者有个人和公司，每个人/公司在贡献的代码文件的时候，在申明里面可以申明特定的许可证的，不一定和开源项目本身声明的保持一致


Leo

发件人: Johnny ***@***.***>
发送时间: Friday, May 13, 2022 12:29 PM
收件人: ***@***.***>
抄送: ***@***.***>
主题: Re: [kaiyuanshe/ONES] Dual licensing 在开源盈利中的作用 (Discussion #87)


有个困惑，关于双 License，我理解的一直都是企业商用协议+普通用户开源协议的混合版本。但是最近看到一些 Apache/MIT, GPL/MIT 的混合 license，这里面是有什么讲究么？

比如说 https://github.com/NVIDIA/open-gpu-kernel-modules 采用的好像是 GPL/MIT

―

**Reply by will-ww** [reply]
这个话题值得好好深入聊下，需要做些功课了。一个组织作为开源策略的设计参考~

**Comment by wangleo61** [comment]
是的，这个我记得中意对这个也挺有研究的，下次请他一起聊聊王老师

Leo

发件人: Will ***@***.***>
发送时间: Saturday, May 14, 2022 6:50 AM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Dual licensing 在开源盈利中的作用 (Discussion #87)


这个话题值得好好深入聊下，需要做些功课了。一个组织作为开源策略的设计参考~

―

**Reply by will-ww** [reply]
> 是的，这个我记得中意对这个也挺有研究的，下次请他一起聊聊

好咧，安排上~

**Comment by wangleo61** [comment]
期待哈，王老师


发件人: Will ***@***.***>
发送时间: Saturday, May 14, 2022 2:12 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] Dual licensing 在开源盈利中的作用 (Discussion #87)


是的，这个我记得中意对这个也挺有研究的，下次请他一起聊聊

好咧，安排上~

―
